src/app.js

```diff
@@ -1,3 +1,12 @@
 'use strict';
 
-// Write your code here
+const { checkIsValidUserInput } = require('./modules/checkIsValidUserInput');
+const { generateRandomNumber } = require('./modules/generateRandomNumber');
+const { getBullsAndCows } = require('./modules/getBullsAndCows');
+
+const numberToGuess = generateRandomNumber();
+const userInput = +process.argv.slice(2)[0];
```

Comment:
The `userInput` is being converted to a number using the unary `+` operator. However, the `checkIsValidUserInput` function expects a string input. Consider removing the `+` to keep `userInput` as a string.